roverrobotics_driver/src/roverrobotics_ros_driver.cpp

```diff
@@ -25,49 +13,57 @@ ROSWrapper::ROSWrapper(ros::NodeHandle *nh) {
     ROS_FATAL("No communication method set, Shutting down Driver Node");
     ros::shutdown();
   }
-  if (!ros::param::get("closed_loop_control", closed_loop)) {
-    ROS_INFO("no 'closed_loop_control' set; using the default value: 'false'");
-    closed_loop = false;
+  std::string mode_name = "OPEN_LOOP";
```

Comment:
It is my opinion that any default values for variables should be defined in a header file.